### ISSUE TITLE
Can't run RegionFixer

### ISSUE THREAD

**Issue by mouri8** [Fenixin/Minecraft-Region-Fixer#117]
Hi! I'm trying to use this tool to see if there's a corrupted chunk or region of my minecraft world because I was having some issues, but I can't run it on cmd.
The problem is that when I type the command in the cmd, nothing happens. I can tell it's NOT the PATH problem some people had, because I can run other .py files. It's just this one that doesn't respond.

> C:\Users\rvila>cd C:\Users\rvila\Desktop\Minecraft-Region-Fixer-0.3.2
> 
> C:\Users\rvila\Desktop\Minecraft-Region-Fixer-0.3.2>py regionfixer.py "C:\Users\rvila\Desktop\cabesas"
> 
> C:\Users\rvila\Desktop\Minecraft-Region-Fixer-0.3.2

As you can see, it returns the same line as the beggining. I know that it can find it because if I type another name instead of regionfixer.py it says that the file doesn't exist.
By the way, I found out there that Python 3.7 for Win10 uses py instead of python as PATH.
**Expected behavior**
It should start to check the world folder (cabesas).
Thanks! I hope there's an easy solution!

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Python version: 3.7 32bit
 - Region Fixer Version: 0.3.2

**Comment by Fenixin** [comment]
Hello!

In my windows 10 with python 3.7.2 I can run a python script using `py` and `python`, both are in the path. I don't know if that has something to do with my version or is the same in everywhere.

The other python scripts that you run, do they print results in the terminal? Have you tried a basic `hello_world.py`? How have you installed you python interpreter?

**Comment by mouri8** [comment]
Hi! Thanks for the quick reply.

I tried to run it on another PC where I downloaded Python 3.8 and run perfectly fine. I don't know about my laptop, but it was working with other .py files and the Python itself.

I don't know if I should open another post because it's a different question, but:
The thing is that when I used the RegionFixer, the scan found one problem on idcounts.dat (which I couldn't understand what was the specific error). Later, the world started working again and I didn't change anything. If you want a better explanation I can open a new post.

Thanks anyway!
